apps/client/src/components/Lessons/LessonInteractiveContent.svelte

```diff
@@ -1,8 +1,8 @@
 <script>
   export let lesson;
-  import CodeEditor from "$components/CodeEditor/CodeEditor.svelte";
-  import EditorTerminal from "$components/CodeEditor/EditorTerminal.svelte";
-  import EditorIFrame from "$components/CodeEditor/EditorIFrame.svelte";
+  import CodeEditor from '$components/CodeEditor/CodeEditor.svelte';
```

Comment:
We shouldn't display this if the lesson is just text.

Reply:
Good catch!